code/game/objects/items/melee/energy.dm

```diff
@@ -55,7 +55,7 @@
 		var/mob/living/carbon/C = user
 		if(C.wear_mask == src)

```

Comment:
While we're here I'm pretty sure this check and the one in the other file are wrong, given that these swords cannot be worn on the face.